mrchecker-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/mrchecker/webapi/core/utils/HTMLParser.java

```diff
@@ -0,0 +1,79 @@
+package com.capgemini.mrchecker.webapi.core.utils;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class HTMLParser {
```

Comment:
Same as in PR#12:
In my opinion, there are too much 'wrapping methods' over already wrapped Jsoup methods. This class will grow very fast -> e.g. to use our Parser for the next HTML element we will need to create 3 additional methods.
Please create more general methods -> only extending features for already existing in jsoup. 
Example:
```
public final class HTMLParser {

	private HTMLParser() {
	}

	public static Document parse(Response response) {
		return Jsoup.parse(response.body().asString());
	}

	public static Document parse(String html) {
		return Jsoup.parse(html);
	}

	public static List<String> getAttributeValueFromElements(Elements elements, String attributeKey) {
		return elements.stream().map(el -> el.attr(attributeKey)).collect(Collectors.toList());
	}

}
```

PS. Please include unit tests for each class which is not Page/Test Class.

Reply:
Same as my response for PR#12

Reply:
Done